### ISSUE TITLE
Dirt Rally view locked to room scale position and orientation

### ISSUE THREAD

**Issue by Raging-Beard** [LibreVR/Revive#1243]
Using latest Revive (tried older ones too) and latest SteamVR (not in Beta).

Dirt Rally launches in VR but the in game menu is locked in position where my desk is, which is of course where I pointed monitor as being in room setup.

Recenter view (L Ctrl) does not re-position the view. This game has always run great in the past and have always had SteamVR in room scale. I tried standing mode anyway, but still L Ctrl has no effect.

When I press L Ctrl the menu flashes infront of me for a millisecond, so fast your eye just catches a ghost image of it, but this shows the game registering the input.

Any ideas please?
Thanks.

**Comment by DaRkL3AD3R** [comment]
Using Revive 1.6.3 for compatibility reasons with Steam version of Dirt Rally, I do not have this issue. Also my SteamVR is beta 1.5.17 and I am using an Index kit.

**Comment by DaRkL3AD3R** [comment]
Amazingly it started happening to me between then and now.

**Comment by Raging-Beard** [comment]
I have fixed mine. So an uninstall and install didn't work first time, but on second go I deleted SteamVR folders which uninstall didn't remove, problem was fixed!

Having contacted Steam Support and sent them my system report, they noted that they could see Revive and ALVR drivers installed. Well I have used and again still use Revive and all is good, so perhaps ALVR was causing the issue.

In case you don't know, ALVR allows SteamVR games to be played wirelessly on the Oculus Quest.
